lib/pure/pathnorm.nim

```diff
@@ -76,6 +76,11 @@ proc addNormalizePath*(x: string; result: var string; state: var int;
       if (state shr 1) >= 1:
         var d = result.len
         # f/..
+        ## We could handle stripping trailing sep here as well: foo// => foo
```

Comment:
Why doc-comments suddenly?

Reply:
that was do distinguish from commented code snippet, but I've now removed it